internal/monitors/diagnostics.go

```diff
@@ -72,19 +72,23 @@ func (mm *MonitorManager) DiagnosticText() string {
 		serviceStats := ""
 		if am.endpoint != nil {
 			serviceStats = fmt.Sprintf(
-				"Discovery Rule: %s\n"+
-					"Monitored Endpoint ID: %s\n",
+				`Discovery Rule: %s
+Monitored Endpoint ID: %s`,
 				am.config.MonitorConfigCore().DiscoveryRule,
 				am.endpoint.Core().ID)
 		}
 		activeMonText += fmt.Sprintf(
-			"%s. %s\n"+
-				"    Reporting Interval (seconds): %d\n"+
-				"%s"+
-				"    Config:\n%s\n",
+			`%s. %s
```

Comment:
This indentation is pretty hard to read.

Reply:
I don't really know how to fix it given `go fmt` and its opinions. The indentation was previously weird as well.